### ISSUE TITLE
Install Istio 1.6-latest via istioctl got unknown filed "certmananger" error 

### ISSUE THREAD

**Issue by carolynhu** [istio/istio#23610]
Install using this release: https://gcsweb.istio.io/gcs/istio-build/dev/1.6-alpha.51aa02587524736ebb7f194f679db08f7620308e/istio-1.6-alpha.51aa02587524736ebb7f194f679db08f7620308e-linux-amd64.tar.gz

Here is the error log:

```
~/prow/go/src/istio.io/tools/perf/istio-install/tmp ~/prow/go/src/istio.io/tools/perf/istio-install
+ ./istioctl install --charts ./manifests -f /home/prow/go/src/istio.io/tools/perf/istio-install/istioctl_profiles/default.yaml --set meshConfig.rootNamespace=istio-system --force=true
Validation errors (continuing because of --force):
unknown field "certmanager" in v1alpha1.Values
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x21f1201]

goroutine 1 [running]:
istio.io/istio/operator/pkg/apis/istio/v1alpha1/validation.validateFeatures(0xc0013015f0, 0xc0005cb200, 0x0, 0x0, 0x0)
	istio.io/istio/operator/pkg/apis/istio/v1alpha1/validation/validation.go:111 +0x51
istio.io/istio/operator/pkg/apis/istio/v1alpha1/validation.ValidateConfig(0x29c0500, 0xc000868630, 0xc0005cb200, 0xb8c, 0xc0006a08a0, 0x56, 0x2e88a01, 0xc0002c1600)
	istio.io/istio/operator/pkg/apis/istio/v1alpha1/validation/validation.go:45 +0x201
istio.io/istio/operator/cmd/mesh.GenerateConfig(0xc0009315b0, 0x1, 0x1, 0xc0006a08a0, 0x56, 0xc000b8f701, 0xc0002c1600, 0x2e88ae0, 0xc000482fc0, 0x0, ...)
	istio.io/istio/operator/cmd/mesh/profile-common.go:62 +0x187
istio.io/istio/operator/cmd/mesh.ApplyManifests(0xc000245500, 0x2, 0x2, 0xc0009315b0, 0x1, 0x1, 0x1, 0x0, 0x0, 0x0, ...)
	istio.io/istio/operator/cmd/mesh/manifest-apply.go:174 +0x196
istio.io/istio/operator/cmd/mesh.runApplyCmd(0xc000a10840, 0xc00052e3e7, 0xc000085400, 0xc000aaa090, 0x0, 0x0)
	istio.io/istio/operator/cmd/mesh/manifest-apply.go:150 +0x279
istio.io/istio/operator/cmd/mesh.InstallCmd.func1(0xc000a10840, 0xc000917c70, 0x0, 0x7, 0x0, 0x0)
	istio.io/istio/operator/cmd/mesh/manifest-apply.go:130 +0x46
github.com/spf13/cobra.(*Command).execute(0xc000a10840, 0xc000917c00, 0x7, 0x7, 0xc000a10840, 0xc000917c00)
	github.com/spf13/cobra@v1.0.0/command.go:842 +0x453
github.com/spf13/cobra.(*Command).ExecuteC(0xc0002602c0, 0x8, 0x8, 0xc0002602c0)
	github.com/spf13/cobra@v1.0.0/command.go:950 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.0.0/command.go:887
main.main()
	istio.io/istio/istioctl/cmd/istioctl/main.go:29 +0x76
```

**Comment by howardjohn** [comment]
This field has not done anything for many releases and was removed

**Comment by esnible** [comment]
Duplicate of https://github.com/istio/istio/issues/23562 and fixed by https://github.com/istio/istio/pull/23587

**Comment by esnible** [comment]
BTW, I tested with `istioctl manifest generate --charts manifests -f ~/go/src/istio.io/tools/perf/istio-install/istioctl_profiles/default.yaml --set meshConfig.rootNamespace=istio-system` to confirm this isn't failing.  I see a different deprecation than you see, I am using the latest istio.io/tools.  If you are using an older one let me know.

```
! global.mtls.enabled is deprecated; use the PeerAuthentication resource instead
```

**Comment by carolynhu** [comment]
> This field has not done anything for many releases and was removed

This error got from the perf pipeline run yesterday, so it must be the latest 1.6 release. Don't know why still has error if this field is deprecated for a while.

**Comment by carolynhu** [comment]
> Duplicate of #23562 and fixed by #23587

@esnible Maybe should backport your fix to 1.6 release also?

**Comment by carolynhu** [comment]
The 1.6-release pipeline failure: https://storage.googleapis.com/istio-prow/logs/daily-performance-benchmark-release-1.6/1/build-log.txt

same issue.
